### PR TITLE
Fix test fails in windows PC

### PR DESCRIPTION
**Pull request by PeterYusuke**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Description
Test output have fails releted windows routing.
I change route string and set TestState as not test class.

### CODE REVIEW

File: tests/test_state.py [hunk 1]
```diff
@@ -1,657 +1,660 @@
...
-    assert getattr(test_state, "dynamic_list") == [5, 10]
-
-    # how to test that one?
-    # test_state.dynamic_list.append(15)
-    # assert test_state.dynamic_list == [5, 10, 15]
-
-    test_state.add_var("dynamic_dict", Dict[str, int], {"k1": 5, "k2": 10})
-    assert test_state.dynamic_dict == {"k1": 5, "k2": 10}
-    assert getattr(test_state, "dynamic_dict") == {"k1": 5, "k2": 10}
+from typing import Dict, List
```
Comment (picklelo) [inline tests/test_state.py:1]: Looks good, but why is the diff so large on this file? (I think there may be some extra carriage returns or some weird thing like that) - can we resolve this

**Review by picklelo — CHANGES_REQUESTED** [review]

File: tests/test_state.py (hunk 1 above)
Comment (FHU-yezi) [reply tests/test_state.py:1]: Line breaks in Windows and Unix-like OS? When I switch from Windows to Linux a few months ago, similar thing happend.

Comment (PeterYusuke) [reply tests/test_state.py:1]: I think this was caused line breaks too.
The diff is only few line at TestState class.

```py
class TestState(State):
    """A test state."""

    # Set this class as not test one   # <- these 2 lines
    __test__ = False                   # <-
```

I think I should revert them before the commit in order to delete line breaks, then commit again.
What do you think?

Comment (Alek99) [reply tests/test_state.py:1]: I think that should work try that, or rebase on the current main branch and commit again

**Comment by PeterYusuke** [comment]
I rebase on the current main branch and change line breaks.
Please check this again.

**Review by picklelo — APPROVED** [review]
Awesome!

**Pull request merged**